printlabels/printlabels.css

```diff
@@ -0,0 +1,149 @@
+body {
+  font-family: Sans-Serif;
+  padding: 0;
+  margin: 0;
+  --label-outline: 0 0 0 1px #aaa;
+}
+
+*, *:before, *:after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.status {
+  padding: 8px;
+  margin-top: 40px;
+  color: blue;
+}
+
+.label-content {
+  white-space: pre-wrap;
+  text-align: left;
+  font-size: 11pt;
+}
+
+.page-outer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 8.5in;
```

Comment:
The sizes actually come from listings like this one https://www.avery.com/blank/labels/94200/gsf-paper (down by Product Specifications). The actual labels that I found that seem popular (and that I've used myself) are 8.5x11. I see there are A4 labels too out there, but no idea what sizes are actually popular enough to support. I am tempted to keep it simpler, but if anyone asks for another size, I'd be open to adding one.

Another issue is that to do labels entirely properly, we ought to be generating a pdf. It turns out still hard to rely on browsers to respect things like print margins. (E.g. if some browser/printer combination insisted on unequal top/bottom margins, the current approach to aligning labels wouldn't work.) But generating a pdf is more work...

Reply:
8.5x11 isn't a thing in Europe (inches aren't even a thing in most of Europe), unless something has changed since I last printed there. I didn't learn about it until academia when wondering why american papers printed funny. But I'm fine being US-centric for now.